scijava/scijava-ops-engine/src/main/java/org/scijava/ops/engine/util/Ops.java

```diff
@@ -82,7 +82,7 @@ private Ops() {
 	 * @return
 	 */
 	public static String[] parseOpNames(String names) {
-		return Arrays.stream(names.split(",")).map(s -> s.trim())
+		return Arrays.stream(names.split("\\s*,\\s*")).map(s -> s.trim())
```

Comment:
Doesn't the `s.trim()` already take care of squelching any whitespace surrounding the commas?

Reply:
...it should. I don't remember why I added it in...

Removed it, though.